conans/test/util/tools_test.py

```diff
@@ -1458,12 +1458,14 @@ def test_repo_root(self):
         svn = SVN(folder=tmp_folder)
         svn.clone(url=project_url)
 
-        self.assertEqual(tmp_folder.replace('\\', '/').lower(), svn.get_repo_root().lower())
+        path = os.path.realpath(tmp_folder).replace('\\', '/').lower()
+        self.assertEqual(path, svn.get_repo_root().lower())
 
         # SVN instantiated in a subfolder
         svn2 = SVN(folder=os.path.join(tmp_folder, 'subdir'))
         self.assertFalse(svn2.folder == tmp_folder)
-        self.assertEqual(tmp_folder.replace('\\', '/').lower(), svn2.get_repo_root().lower())
+        path = os.path.realpath(tmp_folder).replace('\\', '/').lower()
+        self.assertEqual(path, svn2.get_repo_root().lower())
 
```

Comment:
Still the question, if we should deliver on Windows the correct windows path for get_repo_root(). Because for Git module it returns backslashes and for SVN module slashes!

Reply:
I think that SVN always use the concept of a URL (with `file://` protocol if local), so it will always work with forward slashes (and encode spaces and non-ascii chars...)

And Git is using an actual path as the origin for a local repo, so it will use back or forward slashes depending on the platform.

I think that we can leave `svn::get_repo_root` as it is (it returns the output from SVN, the url-like path), because windows can work without any problem with forward slashes.

Reply:
Ok.